scripts/helpers/getlistjson.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports = (context) => {
+  const result = context.map(item => ({
```

Comment:
Nit: can we make this consistent? Either add `()` around `item` or remove them also from `context` above. Also why wrapping the returned object with `()`?

Reply:
First braces can be removed, second braces are required when returning an object from an Arrow function.